index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+    <h1>Hello World, we are software developers!</h1>
+</body>
```

Comment:
- Kindly add a more semantic tags to your HTML document to clearly describe 
-  the meaning of each tag both to the developer and the browser.

- Kindly add a css file to style your html.